tools/dev_qa.py

```diff
@@ -33,7 +33,7 @@
     {
         "name": "buildbuddy",
         "repo_url": "https://github.com/buildbuddy-io/buildbuddy",
-        "commit_sha": "70416ce9f040ba487579dfc773b8f426396bca48",
```

Comment:
Reminder to update here too: https://github.com/buildbuddy-io/buildbuddy-internal/blob/master/.github/workflows/dev-qa.yaml#L37